stetho/src/main/java/com/facebook/stetho/common/android/AccessibilityUtil.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.common.android;
+
+import android.support.v4.view.ViewCompat;
+import android.support.v4.view.accessibility.AccessibilityNodeInfoCompat;
+import android.text.TextUtils;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.widget.AdapterView;
+import android.widget.HorizontalScrollView;
+import android.widget.ScrollView;
+import android.widget.Spinner;
+
+import java.util.List;
+
+/**
+ * This class provides utility methods for determining certain accessibility properties of
+ * {@link View}s and {@link AccessibilityNodeInfoCompat}s. It is porting some of the checks from
+ * {@link com.googlecode.eyesfree.utils.AccessibilityNodeInfoUtils}, but has stripped many features
+ * which are unnecessary here.
+ */
+public final class AccessibilityUtil {
+  private AccessibilityUtil() {
+  }
+
+  /**
+   * Returns whether the specified node has text or a content description.
+   *
+   * @param node The node to check.
+   * @return {@code true} if the node has text.
+   */
+  public static boolean hasText(AccessibilityNodeInfoCompat node) {
```

Comment:
Since `null` is allowed, please add `@Nullable` to the parameter. (same for any other occurrences here -- I'll spare you the repetition)